routes/publish.js

```diff
@@ -0,0 +1,71 @@
+var request = require("request");
+
+function isEmpty(val) {
+  return val === null || val === undefined;
+}
+
+module.exports = function(config) {
+  return function(req, res) {
+    var project = JSON.parse(JSON.stringify(req.session.project.meta));
+    delete project.id;
+    delete project.publish_url;
+
+    if(!req.body || isEmpty(req.body.description) || isEmpty(req.body.dateUpdated) || isEmpty(req.body.public)) {
+      res.status(400).send({ error:  "Request body missing data" });
+      return;
+    }
+
+    project.description = req.body.description;
+    // Uncomment the line below once https://github.com/mozilla/publish.webmaker.org/issues/98 is done
+    // project.public = req.body.public;
+    project.date_updated = req.body.dateUpdated;
+
+    request({
+      method: "PUT",
+      uri: config.publishURL + "/projects/" + req.session.project.meta.id,
+      headers: {
+        "Authorization": "token " + req.user.token
+      },
+      body: project,
+      json: true
+    }, function(err, response, body) {
+      if(err) {
+        res.status(500).send({error: err});
+        return;
+      }
+
+      if(response.statusCode !== 201) {
```

Comment:
I'd also combine these two cases under one response code, unless there's a reason you need the publish server's response back.